modules/test/playwright/utils/showUI.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, expect} from '@playwright/test';
+
+export async function showUI({
+	autoClick,
+	target,
+	trigger,
+}: {
+	autoClick?: Boolean;
+	target: Locator;
+	trigger: Locator;
+}) {
+	const timeout = 100;
+
+	await expect(async () => {
+		await trigger.click();
+		await expect(target).toBeVisible({timeout});
+
+		if (autoClick && (await target.isVisible({timeout}))) {
+			await target.click();
+		}
+	}).toPass();
```

Comment:
IMO (also tested) the IF can be simplified to: `if (autoClick)`

Reply:
Also, I would change the name of the function to something more specific like `clickAndExpectToBeVisible` or even `retryingClickAndExpectToBeVisible` or `clickAndExpectToBeVisibleWithRetry`....

Reply:
Agree with @marcogalluzzi. 
Also, another question, in all cases we are setting` autoClick: true`. Would there be a case where make sense to have it to `false`? Could you give me an example? I guess it is general because we put it as a util, so everyone can use it, not only for KB.

Reply:
> IMO (also tested) the IF can be simplified to: `if (autoClick)`

But it is strange, if we do not specify the timeout in the click, the default one will be used and we lose the quick retry at least in theory 🤔

Reply:
> Also, I would change the name of the function to something more specific like `clickAndExpectToBeVisible` or even `retryingClickAndExpectToBeVisible` or `clickAndExpectToBeVisibleWithRetry`....

I like `clickAndExpectToBeVisible` but I wouldn't put the retry on the name. The retry strategy is a detail of the implementation.

@marcogalluzzi if `clickAndExpectToBeVisible` is good enough for you I can rename the function 😄

Reply:
> Would there be a case where make sense to have it to `false`? 

Yes, it is intentional. I prefer to have it explicit because I feel more semantic but It is only my personal opinion 😄 

> Could you give me an example? I guess it is general because we put it as a util, so everyone can use it, not only for KB.

Yes, for example, expand a section to fill an input, and expand complex UIs like a calendar, or forms. 

Yep I moved to utils because it's needed in some files

Reply:
I remove the `(await target.isVisible({timeout})` part because isn't working as expected tested with @marcogalluzzi. 

With or without `(await target.isVisible({timeout})` the `await target.click();` the click is tried.